tests/Doctrine/Tests/TestUtil.php

```diff
@@ -85,6 +76,10 @@ private static function initializeDatabase(): void
 
         $testConn = DriverManager::getConnection($testConnParams);
 
+        // Note, writes direct to STDERR to prevent phpunit detecting output - otherwise this would cause either an
+        // "unexpected output" warning or a failure on the first test case to call this method.
+        fwrite(STDERR, sprintf("\nUsing DB driver %s\n", get_class($testConn->getDriver())));
```

Comment:
The driver is a required part of the test configuration. Why does it need to be logged?

Reply:
So that in CI (for example if moving to a new platform / changing configuration structure) there is some way to verify that a given job is running tests with the expected driver. Although the risk of this is much lower now that the fallback option has been removed, it may still be possible (e.g. if a matrix variable somehow selects the wrong phpunit config file).

Given the potential consequences of e.g. only running the sqlite memory tests in CI for a prolonged period, it feels worth the extra line of console output to provide a bit more safety / visibility that the configuration has been applied as expected.

Reply:
This way, it would make sense to log all the connection details, not just the driver. Otherwise, if some of the parameters are misspelled, they will be silently ignored.

Reply:
I'm not convinced that logging all the details brings any benefit.

* If the ignored parameters were required to make the connection at all (e.g. it hasn't been given / has a default db host it can't connect to), the driver will throw
* If they were required post-connection to run a test (e.g. an explicit dbname) that test will fail
* If they weren't required for anything, their absence is not relevant

I suppose if it was common to run in an environment with multiple different servers all running at once (e.g. you have mysql 5.7 on one port and mysql 8 on another, and two different phpunit.xml files to connect to the appropriate version) there'd be a case for knowing you were connected to the right port. But that would be an unusual setup, and a long way from how CI is configured at the moment.

So the only silent failure case I can really see is if you're using a completely different driver to the expected one - which is what was happening in the ORM CI.

IMHO therefore there's no benefit to the extra verbosity of logging all the connection details, and it makes it harder to spot the one that matters.